Sources/SwiftProtobuf/JSONEncodingOptions.swift

```diff
@@ -22,5 +22,8 @@ public struct JSONEncodingOptions {
   /// By default they are converted to JSON(lowerCamelCase) names.
   public var preserveProtoFieldNames: Bool = false
 
+  /// Whether to include proto field with default value
+  public var includeDefaultValue: Bool = false
```

Comment:
If I'm reading the code right, this only impacts proto3 syntax files, yes?  (does that match the other languages that have this option)? If so, it should be documented as such (and likely renamed to better capture what it means)